kalite/topic_tools/content_models.py

```diff
@@ -416,19 +429,19 @@ def recurse_availability_up_tree(node, available):
                     recurse_availability_up_tree(parent, available)
 
 
-            for path, update in updates_dict.iteritems():
+            for path, update in content_models:
                 if update:
                     # We have duplicates in the topic tree, make sure the stamping happens to all of them.
-                    items = Item.select().where(Item.path == path)
+                    items = Item.select().where((Item.path == path) & (Item.kind != "Topic"))
+
                     for item in items:
-                        if item.kind != "Topic":
-                            item_data = unparse_model_data(model_to_dict(item, recurse=False))
-                            item_data.update(update)
-                            item_data = parse_model_data(item_data)
-                            for attr, val in item_data.iteritems():
-                                setattr(item, attr, val)
-                            item.save()
-                            recurse_availability_up_tree(item, update.get("available", False))
+                            #item_data = unparse_model_data(model_to_dict(item, recurse=False))
```

Comment:
@rtibbles we removed this section as we assumed that the only thing we need to update here is the availability, not the rest of the fields.

Come to think of it, we don't have the `update` call here @gwong89.